Sources/Rendering/Core/ScalarBarActor/index.d.ts

```diff
@@ -175,6 +180,11 @@ export interface vtkScalarBarActor extends vtkActor {
 	 */
 	setAutoLayout(autoLayout: any): boolean;
 
+  /**
+   * 
```

Comment:
please add more comments about the expected function behavior

Reply:
How does this look to you?

``` javascript
/**
 * Sets the function used to generate legend ticks. 
 * 
 * This function takes a vtkScalarBarActorHelper and returns true on success. 
 * To have the desired effect, the function must call: `helper.setTicks(ticks: num[])` and `helper.setTickStrings(tickStrings: string[])`.
 * 
 * After setting the generateTicks function you must regenerate the vtkScalarBarActor for your changes to take effect. 
 * One way to do that is:
 * ```
 *  const mapper = scalarBarActor.getMapper()
 *  if (mapper) {
 *    mapper.getLookupTable().resetAnnotations()
 *  }
 * ```
 * @param generateTicks 
 */
   ```

Reply:
This is great, thanks.

You also need to give a type to the `generateTicks` parameter

Reply:
Just added a type for this parameter. I'd like to add a type for the vtkScalarBarActorHelper as well but will have to do that at a later date.